client/my-sites/stats/feedback/index.tsx

```diff
@@ -10,8 +15,37 @@ const FEEDBACK_ACTION_LEAVE_REVIEW = 'feedback-action-leave-review';
 const FEEDBACK_ACTION_SEND_FEEDBACK = 'feedback-action-send-feedback';
 const FEEDBACK_ACTION_DISMISS_FLOATING_PANEL = 'feedback-action-dismiss-floating-panel';
 
+const FEEDBACK_PANEL_PRESENTATION_DELAY = 3000;
 const FEEDBACK_LEAVE_REVIEW_URL = 'https://wordpress.org/support/plugin/jetpack/reviews/';
 
+const FEEDBACK_SHOULD_SHOW_PANEL_API_KEY = NOTICES_KEY_SHOW_FLOATING_USER_FEEDBACK_PANEL;
+const FEEDBACK_SHOULD_SHOW_PANEL_API_HIBERNATION_DELAY = 30; // 30 seconds for now
```

Comment:
Currently only 30 seconds to make this easy to test. Will update the value to 6 months before merging the PR.

Reply:
Addressed here:

https://github.com/Automattic/wp-calypso/pull/94398/commits/bf94be1f3a094fe348b1a77bb058a52688ee718f